src/Uno.UI/UI/Xaml/DependencyProperty.Dictionary.TypeToProperties.cs

```diff
@@ -10,11 +11,11 @@ public sealed partial class DependencyProperty
 	{
 		private class TypeToPropertiesDictionary
```

Comment:
This dictionary has a single static instance that is kept for the lifetime of the whole app.